### PR TITLE
osbuilder: add description for how to use DISTRO variable

### PR DESCRIPTION
**Pull request by liubin**
Fixes: #1312

Signed-off-by: bin <bin@hyper.sh>

### CODE REVIEW

**Comment by fidencio** [comment]
I'm really sorry for the disruption but in order to do the branch name changes announced here I'm adding the "do-not-merge" and the "needs-branch-adjustment" labels to the PR.

I'll add a new comment once the branch changes are done and once it happens I'd like to ask you to resubmit the PR against the new main branch, possibly linking to this PR in case you find the conversations / reviews already made important to proceed.

**Comment by fidencio** [comment]
Kata Containers repos are back to business.  Thanks a lot for your patience and I'm very happy to announce that there's no changes needed from your side as renaming the branches automagically reflected here.

If you notice something weird, some missing CI, please, do not hesitate on contacting us here or on #kata-dev on bit.ly/katacontainersslack.

Last but not least, this is a good opportunity to have your PR rebased atop of `main`, to ensure your PR follow all the guidelines (https://github.com/kata-containers/community/blob/master/CONTRIBUTING.md#patch-format) enforced by this community, or even to re-evaluate whether the PR is currently needed.

Happy hacking!

File: tools/osbuilder/README.md [hunk 1]
```diff
@@ -95,6 +95,9 @@ filesystem components to generate an initrd.
 3. When generating an image, the initrd is extracted to obtain the base rootfs for
 the image.
 
+The default distro used to build a rootfs is CentOS, to use a different one, you can set `DISTRO=<your_distro>`.
+For example `make USE_DOCKER=true DISTRO=ubuntu rootfs` will make Ubuntu rootfs using Docker.
```
Comment (Tim-Zhang) [inline tools/osbuilder/README.md:99]: How about `CentOS is the default distro for building the rootfs to use ...`

Comment (Tim-Zhang) [reply tools/osbuilder/README.md:99]: or just `used to` -> `is used to`

Comment (wainersm) [reply tools/osbuilder/README.md:99]: I liked that suggestion. Other than that, looks good to me.

**Review by cmaf — APPROVED** [review]

**Review by Tim-Zhang — APPROVED** [review]

**Pull request merged**